thoth_user_api/utils.py

```diff
@@ -331,3 +332,23 @@ def get_cronjob(cronjob_name: str) -> dict:
         _LOGGER.error(response.text)
     response.raise_for_status()
     return response.json()
+
+
+def logger_warning(function):
+    """Decorator function which sets the logger to warning only mode."""
+    @wraps(function)
+    def wrapper_func():
+        logging.getLogger('werkzeug').setLevel(logging.WARNING)
```

Comment:
It is because the logger which prints the messages in the openshift are from flask. This logger werkzeug is a logger of flask and when we set the logging level of this logger to some level as required we can minimise the logs getting printed for this.